pkg/dom0-ztools/rootfs/etc/init.d/001-getty

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# we override content of https://github.com/linuxkit/linuxkit/blob/master/pkg/getty/etc/init.d/001-getty
+# with conditional behavior
+# if we see getty in kernel command line
+# we will start getty
+
+if cat /proc/cmdline | grep -q getty; then
+  export INITGETTY=true
+  export INSECURE=true
+  /usr/bin/rungetty.sh
```

Comment:
With this, do we still need/use linuxkit/getty:v0.5?

Reply:
The main idea here is to keep getty in init section to be able to debug. I put an option `enable getty` in grub menu to enable it for debugging.
Also we still rely on files and logic available in `linuxkit/getty:v0.5` from pillars part of code (I put the comment into `rootfs.yml.in` about it)